python/leap/docopt.py

```diff
@@ -0,0 +1,581 @@
+"""Pythonic command-line interface parser that will make you smile.
+
+ * http://docopt.org
+ * Repository and issue-tracker: https://github.com/docopt/docopt
+ * Licensed under terms of MIT license (see LICENSE-MIT)
+ * Copyright (c) 2013 Vladimir Keleshev, vladimir@keleshev.com
+
+"""
```

Comment:
Yes, but for some reason docopt isn't playing well with my computer. I get "no module" errors or it returns __doc__ in python 2 and 3 respectively. I haven't been able to fix it.